roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,41 @@
+# pylint: disable=missing-docstring
+
+"""Interface to OpenShift oc command"""
+
+import shlex
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    module = AnsibleModule(
```

Comment:
Have we considered moving the checks from 'oadm diagnostics AggregatedLogging' to here.  I think they would make more sense to live in something that will change quicker then oadm